CSharp/Library/Microsoft.Bot.Connector.Shared/StateAPI/Models/BotData.cs

```diff
@@ -21,9 +21,8 @@ public partial class BotData
         /// <summary>
         /// Initializes a new instance of the BotData class.
         /// </summary>
-        public BotData(string eTag = default(string), object data = default(object))
+        public BotData(string eTag = @"*", object data = default(object))
```

Comment:
@RobStand how can we file an issue to remove this Etag star from the documentation?  It is incorrect guidance because it can cause data loss.